tests/system/CMakeLists.txt

```diff
@@ -336,7 +343,28 @@ ENDIF()
                     TRUE)
 
   IF(NOT QMC_CUDA)
+
+# Check VMCUpdateAllWithDrift driver using all-particle updates
+  QMC_RUN_AND_CHECK(short-bccH_1x1x1_ae-vmc-all_sdj
+                    "${CMAKE_SOURCE_DIR}/tests/solids/bccH_1x1x1_ae"
+                    qmc_short_vmc_all
+                    qmc_short_vmc_all.in.xml
+                    1 16
+                    BCC_H_SCALARS
+                    0 # VMC
+                    TRUE)
   LIST(APPEND BCC_H_RMC_SCALARS "totenergy" "-1.8420 0.0011")
```

Comment:
It looks like the DMC test got stuck between the RMC energy definition and the RMC test.